base/Dockerfile

```diff
@@ -1,5 +1,13 @@
 FROM ubuntu:18.04
 
+# Runner user parameters
+ARG UNAME=runner
+ENV UNAME=$UNAME
+ARG UID=1000
+ENV UID=$UID
+ARG GID=1000
+ENV GID=$GID
```

Comment:
Do we have to expose these args or can we just use ENV only for these?

Reply:
If we only keep one or the other, it's better to keep ARG, this is a build-time use-case, and only `base` actually needs it

I kept the ENV in case people want to be able to do something like
```Dockerfile
USER 0:0
RUN some --root-commands
USER $UID:$GID
RUN cat regular-user.file
```